static/js/buddy_data.js

```diff
@@ -250,13 +250,15 @@ export function get_title_data(user_ids_string, is_group) {
     // a single, human, user.
     const active_status = presence.get_status(user_id);
     const last_seen = user_last_seen_time_status(user_id);
+    const is_user_you = page_params.user_id === user_id;
```

Comment:
I am not sure what to name this variable?
Maybe `is_user_me` or `is_user_you` ?

Reply:
In `popovers.js`, `is_me` is used for a similar purpose. 
Here is the line : `const is_me = people.is_my_user_id(user.user_id);`
Maybe `is_me` will be more appropriate here.

Reply:
Using `is_me` seems like the better option.
Thanks, @smitpatel24.